release-notes-3.4.0.md

```diff
@@ -1,8 +1,7 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
-
+- [SCMOD-8955]: Updated worker framework version
```

Comment:
Not formatted correctly, look at other examples.

Reply:
Done